ppr-ui/.eslintrc.js

```diff
@@ -2,15 +2,17 @@ module.exports = {
   root: true,
   env: {
     node: true,
-    es2022: true
+    es2022: true,
   },
   plugins: ['vue'],
   extends: [
     'plugin:vuetify/base',
     'plugin:vue/vue3-recommended',
-    '@vue/typescript/recommended'
+    '@vue/typescript/recommended',
```

Comment:
We are now using trailing commas?

Reply:
Kind of funny, the linter didn't lint it's own config file.